packages/http/src/__tests__/http-prism-instance.spec.ts

```diff
@@ -269,24 +259,7 @@ describe('Http Client .process', () => {
       });
 
       // TODO: will be fixed by https://stoplightio.atlassian.net/browse/SO-259
-      xit('with invalid body returns validation errors', () => {
-        return expect(
-          prism.process({
-            method: 'get',
-            url: {
-              path: '/pet/findByStatus',
-              query: {
-                status: ['available'],
-              },
-            },
-            body: {
-              id: 'should not be a string',
-              status: 'should be one of "placed", "approved", "delivered"',
-              complete: 'should be a boolean',
-            },
-          }),
-        ).rejects.toThrowError(ProblemJsonError.fromTemplate(UNPROCESSABLE_ENTITY));
-      });
+      test.todo('with invalid body returns validation errors');
```

Comment:
:fire: :fire: :fire:  You're removing my lovely tests :( :fire:

Reply:
It was skipped — I'm fine putting it back but it's not getting executed anyway. I'd rather see a todo that says "Hey write this" instead of a skip that you do not easily see